tasks/arrays/arrays.js

```diff
@@ -1,62 +1,94 @@
-/**
- * 1. Write a method to reverse a string; Function should return the string 'This is not a string'
- * if a number is passed
-* @param {string}
-*/
-function reverseString(str) {
-}
-
 /**
  * return an array of characters names
+ * @param {Array} chars
+ * @return {Array} - char names
  */
-function getCharactersNames(chars) {
+function getCharactersNames(chars, ) {
+  let result = chars.map(({ name }) => name);
+  return result;
 }
 
 /**
- * print (console.log) ids of all characters
- */
+* print (console.log) ids of all characters
+* @param {Array} chars
+*/
 function printCharacterNames(chars) {
+  let result = chars.map(({ name }) => name);
+  return result;
+
 }
 
 /**
- * return an array of non-human characters
- */
+* return an array of non-human characters
+* @param {Array} chars
+* @return {Array} - non human characters
+*/
 function getNonHumanCharacters(chars) {
+  let result = chars.filter(obj => {
+      return obj.species !== 'Human'
+    });
+    let newArray = Array.from(result);
```

Comment:
this is too complex, you do not need to use Array.from(...);
just simply return your result;